### PR TITLE
Enable textAlignVertical for textbox

### LINKED ISSUE #269: textAlignVertical not supported for textInput
### Version

- tcomb-form-native v0.6.1
- react-native v0.36.1

### Expected behaviour

According to the documentation, it is possible to use the form option `textAlignVertical`.

### Actual behaviour

`textAlignVertical` doesn't work, it is not taken into account by the library

### Steps to reproduce

Create a form with a field of type `t.String`. Enable multiple line, modify the width of the corresponding field (for the sake of the example, let's say 100). As of now, when the user write a text, the text will be inserted in a vertically-centred manner. When using the option `textAlignVertical: 'top'`, no change can be observed.

### PR DESCRIPTION
**Pull request by jeromegrosse**
Fix #269

### CODE REVIEW

**Comment by gcanti** [comment]
Thanks @jeromegrosse, @alvaromb may I merge?

**Comment by alvaromb** [comment]
I can't find `textAlignVertical` in the `TextInput` Props. Seems to be applied to the `style` @jeromegrosse in the latest RN version.

**Comment by jeromegrosse** [comment]
@gcanti @alvaromb Thank you both for your answer

I believe that it is a problem with the documentation of React Native. However, they definitely display an example with `textAlignVertical` [here](http://facebook.github.io/react-native/docs/textinput.html).

If you search for `textAlignVertical`, you'll find the following snippet:
```js
<TextInput
   autoCorrect={true}
   multiline={true}
   style={[styles.multiline, {color: 'blue'}, {textAlign: 'right', textAlignVertical: 'bottom'}]}>
   <Text style={styles.multiline}>multiline with children, aligned bottom-right</Text>
 </TextInput>
```

It can also be found in [UIExplorer](https://github.com/facebook/react-native/blob/e8198aed8d094087d5da898a325916dd44b86d5a/Examples/UIExplorer/js/TextInputExample.android.js#L596).

**Comment by alvaromb** [comment]
Yes @jeromegrosse, you're right that it is in the docs, but not as a prop (as your PR uses), it's displayed as a property of the `style` prop. You need to change your PR and pass the `textAlignVertical` through the style.

File: lib/templates/bootstrap/textbox.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ function textbox(locals) {
         keyboardType={locals.keyboardType}
         maxLength={locals.maxLength}
         multiline={locals.multiline}
+        textAlignVertical={locals.textAlignVertical}
```
Comment (alvaromb) [inline lib/templates/bootstrap/textbox.js]: `locals.textAlignVertical` should be passed in the `style` prop.

**Comment by jeromegrosse** [comment]
Oh, you are definitely right. Strangely enough, passing it `textAlignVertical` to `TextInput` as property also work. I guess that it has to do with how `TextInput` works and pass this parameter to `Text`.

I guess that this PR is therefore not needed.

**Comment by gcanti** [comment]
> I guess that this PR is therefore not needed

Ok, thanks @jeromegrosse. Closing for now

**Comment by laylawu2** [comment]
I was able align text to the top of the textbox on my android screen by doing: 
const TForm = t.form.Form;
const textAreaStyle = _.cloneDeep(TForm.stylesheet);
textAreaStyle.textbox.normal.textAlignVertical = 'top';

**Pull request closed without merging**